### PR TITLE
Support struct_element_addr instruction

### PR DESCRIPTION
**Pull request by mmroz**

### CODE REVIEW

File: lib/WALASupport/SILWalaInstructionVisitor.cpp [hunk 1]
```diff
@@ -1010,6 +1010,32 @@ jobject SILWalaInstructionVisitor::visitStructInst(StructInst *SI) {
...
+        llvm::outs() << "\t [OPERAND ADDR]: " << StructOperand.getOpaqueValue() << "\n";
+        llvm::outs() << "\t [STRUCT]: " << StructElement->getDeclaredType().getString() << "\n";
+        llvm::outs() << "\t [STRUCT FIELD]: " << StructField->getNameStr() << "\n";
+  }
+
+  string FieldName = StructField->getNameStr();
+  jobject FieldNameNode = Wala->makeConstant(FieldName.c_str());
+  jobject FieldNode = Wala->makeNode(CAstWrapper::VAR, FieldNameNode);
+
+  auto Node = Wala->makeNode(CAstWrapper::OBJECT_REF, ElementNode , FieldNode );
```
Comment (karimhamdanali) [inline lib/WALASupport/SILWalaInstructionVisitor.cpp]: Please cleanup spacing before the closing parenthesis.

**Comment by taufique71** [comment]
@mmroz Just asking, what does `OBJECT_REF` node take? Is there anything specific?

**Comment by mmroz** [comment]
A reference to an object

**Comment by mmroz** [comment]
Fixed the space!

**Review by karimhamdanali — APPROVED** [review]

**Pull request merged**